cpp/CMakeLists.txt

```diff
@@ -710,18 +714,6 @@ if(TARGET conda_env)
   target_link_libraries(cudf PRIVATE conda_env)
 endif()
 
-if(CUDA_STATIC_RUNTIME)
```

Comment:
We will still need
```cmake
if(CUDA_STATIC_RUNTIME AND NOT BUILD_SHARED_LIBS )
  # Make sure to export to consumers what runtime we used
  target_link_libraries(cudf PUBLIC CUDA::cudart_static)
endif()

since when doing static linking libcudf won't have linked to the cuda runtime and so we need to tell the consumer that is the intent.